src/Coalition/Api/Listener/PreEventEditListener.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Coalition\Api\Listener;
+
+use ApiPlatform\Core\EventListener\EventPriorities;
+use App\Entity\Event\BaseEvent;
+use App\Event\EventEvent;
+use App\Events;
+use App\Repository\Event\BaseEventRepository;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+class PreEventEditListener implements EventSubscriberInterface
```

Comment:
this listener (like `PostEventEditListener.php`) seems to work for all Events, not only Coalition/Cause. Could you move these 2 listeners in another place, please?